applications/GeoMechanicsApplication/custom_strategies/strategies/geo_mechanics_newton_raphson_erosion_process_strategy.hpp

```diff
@@ -305,41 +274,32 @@ class GeoMechanicsNewtonRaphsonErosionProcessStrategy
     }
 
     template <typename FilteredElementType>
```

Comment:
For consistency with the new changes, you may want to rename the template parameter:
```suggestion
    template <typename FilteredElementsType>
```

Reply:
yes done